backend/cellguide/pipeline/__init__.py

```diff
@@ -65,18 +65,27 @@ def upload_cellguide_pipeline_output_to_s3(*, output_directory: str):
     the CellGuide snapshot to the corresponding environment's CellGuide data bucket.
     """
     deployment_stage = os.getenv("DEPLOYMENT_STAGE")
+    remote_dev_prefix = os.getenv("REMOTE_DEV_PREFIX")
     s3_provider = S3Provider()
 
     if not deployment_stage:
         logger.warning(f"Not uploading the pipeline output at {output_directory} to S3 as DEPLOYMENT_STAGE is not set.")
         return
 
-    if deployment_stage in ["dev", "staging", "prod"]:
+    if deployment_stage == "rdev" and remote_dev_prefix is None:
```

Comment:
The bucket path for rdev deployments is `s3://cellguide-data-public-dev/env-rdev-cellguide` which needs to be suffixed by the rdev stack name.